src/tasks/dev_prerun.js

```diff
@@ -24,49 +20,15 @@ async function registerAdminAccount(dataModelEngine, logger) {
   }
 }
 
-async function addToKycWhitelist(dataModelEngine, logger) {
+export async function addToKycWhitelist(roleIndex, dataModelEngine, logger) {
   logger.info(`Whitelist node address with registry`);
   const address = dataModelEngine.identityManager.nodeAddress();
   const kycContract = await dataModelEngine.contractManager.kycWhitelistContract();
-  const HERMES_ROLE_INDEX = 2;
   await kycContract
     .methods
-    .add(address, HERMES_ROLE_INDEX)
+    .add(address, roleIndex)
     .send({
       from: address,
       gas: 2000000
     });
 }
-
-async function onboardAsHermes(dataModelEngine, logger) {
-  logger.info(`Onboarding node as HERMES`);
-  const address = dataModelEngine.identityManager.nodeAddress();
-  const rolesContract = await dataModelEngine.contractManager.rolesContract();
-  const url = `localhost:${config.serverPort}`;
-  await rolesContract
-    .methods
-    .onboardAsHermes(url)
-    .send({
-      from: address,
-      gas: 2000000
-    });
-}
-
-async function setupDevelopment(dataModelEngine, logger) {
-  await registerAdminAccount(dataModelEngine, logger);
-  await addToKycWhitelist(dataModelEngine, logger);
-  await onboardAsHermes(dataModelEngine, logger);
-}
-
-const builder = new Builder();
-const logger = new WinstonConsoleLogger();
-
-builder.build(config)
-  .then(async ({client, dataModelEngine}) => {
-    await setupDevelopment(dataModelEngine, logger);
-    await client.close();
-  })
-  .catch((exception) => {
-    logger.error(exception);
-    process.exit(1);
-  });
```

Comment:
Why does this file still exist? Node can be onboarded as Hermes or as Atlas. There is no way to onboard as nobody.
As I understand, this functions are used in `dev_prerun_atlas` and `dev_prerun_hermes`, but in this case they definitely should not be placed in `/tasks`.